src/pages/css/properties/color-property/index.md

```diff
@@ -1,15 +1,47 @@
----
-title: Color Property
----
-## Color Property
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/css/properties/color-property/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+## CSS Color Propery 
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+ 
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+You can use the `color` property to set the color of the text in an element.  
 
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+ 
+
+You can use several methods to declare a color such as: 
+* By Name (Note: this only works with certain colors)
+    ```
+    h1{
+        color: blue;
+    }
+    ```
+
+* Hexadecimal (specified as #rrggbb)
+
+    ``` 
```

Comment:
We can add syntax highlighting by starting the code block off stating the language i.e.

` ```css`